site/src/pages/CreateWorkspacePage/CreateWorkspacePageView.tsx

```diff
@@ -112,133 +120,143 @@ export const CreateWorkspacePageView: FC<CreateWorkspacePageViewProps> = ({
   );
 
   return (
-    <FullPageHorizontalForm title="New workspace" onCancel={onCancel}>
-      <HorizontalForm onSubmit={form.handleSubmit}>
-        {Boolean(error) && <ErrorAlert error={error} />}
-        {/* General info */}
-        <FormSection
-          title="General"
-          description="The template and name of your new workspace."
-        >
-          <FormFields>
-            <SelectedTemplate template={template} />
-            {versionId && versionId !== template.active_version_id && (
-              <Stack spacing={1} className={styles.hasDescription}>
-                <TextField
-                  disabled
-                  fullWidth
-                  value={versionId}
-                  label="Version ID"
-                />
-                <span className={styles.description}>
-                  This parameter has been preset, and cannot be modified.
-                </span>
-              </Stack>
-            )}
-            <TextField
-              {...getFieldHelpers("name")}
-              disabled={creatingWorkspace}
-              onChange={onChangeTrimmed(form)}
-              autoFocus
-              fullWidth
-              label="Workspace Name"
-            />
-          </FormFields>
-        </FormSection>
+    <>
+      {mode === "duplicate" && <DuplicateWarningMessage />}
```

Comment:
The only thing that changes with this chunk of code is that the fragment got added, so I could squeeze in the `DuplicateWarningMessage` component. Everything else is the same – it just changed by an indentation level

Reply:
you could also put it next to the `{Boolean(error) ...` and that might look nice